src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.5.td.json

```diff
@@ -0,0 +1,56 @@
+{
+  "_guid_" : "S-131.5",
+  "_extends_" : "F-131_Test_Data_Base",
+
+  "specs" : [
+    "with no U access for the CaseType",
+    "attempts to create an event for the previously created case"
+  ],
+
+  "request": {
+    "_extends_": "Common_Request",
+    "pathVariables": {
+      "caseId": "${[scenarioContext][childContexts][F-131.5_CreateCase][testData][actualResponse][body][id]}"
+    },
+
+    "headers": {
+      "experimental": true
+    },
+    "body": {
+      "data": {
+        "TextField" : "value"
+      },
+      "event": {
+        "id" : "event1"
+      },
+      "ignore_warning": false,
+      "event_token": "${[scenarioContext][childContexts][S-131.5_Token_Creation][testData][actualResponse][body][token]}"
```

Comment:
we can remove this line as it's in the base file

Reply:
Removed this line as its in the base file